sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -783,7 +837,7 @@ def _plot_ice_lines(
         ax,
         pd_plot_idx,
         n_total_lines_by_plot,
-        individual_line_kw,
+        ice_lines_kw,
```

Comment:
wenn du argumente umbenennst bringst du breaking changes in die API, das sollte möglichst vermieden werden, also wenn du keinen super triftigen grund hast, lass es beim alten namen

Reply:
Okay, mein Grund war, dass das ja nach den Änderungen nicht mehr für die individual line, sondern für alle lines ist. Aber ich denke mal das ist nicht triftig genug